app/views/todos/index.html.erb

```diff
@@ -35,7 +36,35 @@
     </div>
 
     <div>
-      <h3>Issues (<%= @my_issues.size %>): </h3>
+      <h4> (<%= @owned_pulls.size %>) Opened by me: </h4>
+      <table class="table">
```

Comment:
maybe we can move this table into a partial an reuse it also for the `@my_pulls` list, I think it's the same table